2020/_includes/footer.html

```diff
@@ -42,33 +42,42 @@ <h2 style="color: #ffffff;">
                                     </div>
                                 </div>
 
-                                <div class="wpb_column vc_column_container vc_col-sm-3">
+                                <div class="wpb_column vc_column_container vc_col-sm-2">
                                 </div>
 
 
-                                <div class="wpb_column vc_column_container vc_col-sm-3">
+                                <div class="wpb_column vc_column_container vc_col-sm-4">
```

Comment:
expanded the section so all icons are in a single row

Reply:
how does it look in mobile? I know the site is not perfect, but just want to avoid new issues

Reply:
it looks like this:
<img width="257" alt="Captura de pantalla 2019-12-30 a las 14 19 04" src="https://user-images.githubusercontent.com/100741/71583623-8d19bf00-2b0f-11ea-91e4-d1d8bb7f2305.png">

Reply:
perfect! :+1: